src/runner.js

```diff
@@ -183,7 +183,7 @@ assign(Runner.prototype, {
   ensureConnection() {
     return Promise.try(() => {
       return this.connection || new Promise((resolver, rejecter) => {
-        this.client.acquireConnection()
+        return this.client.acquireConnection()
```

Comment:
This place needs comment to code, which says something like "need to return promise or null from handler to prevent warning from bluebird"

Otherwise this return statement might be removed by accidents, since it doesn't serve any other purpose here.